dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,10 +1,43 @@
 class RubricsController < ApplicationController
+  include Rails.application.routes.url_helpers
+
   def edit
     @rubric = Rubric.find_by(id: params[:id])
     @lesson = @rubric.lesson
   end
 
   def new
     @lesson = Lesson.find_by(id: params[:lessonId])
+    @rubric = Rubric.new
+  end
+
+  def create
```

Comment:
When chatting with Tyrone, Mark, and Eric the other day, I realized I've gotten lazy on a convention to add the corresponding route above controller actions. Do you mind doing that in this controller? The [reference_guides_controller](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/reference_guides_controller.rb) does this really well as an example.

For example, I believe the route for the `create` action is `POST /rubrics`. For `new`, it's `GET /rubrics/new`. etc